sdk/identity/identity/src/msal/nodeFlows/msalPlugins.ts

```diff
@@ -206,6 +218,18 @@ function generatePluginConfiguration(options: MsalClientOptions): PluginConfigur
           ].join(" "),
         );
       }
+      // There is a scenario where the VSCode plugin is configured but the broker is not available (e.g., missing native dependencies)
+      // This check ensures we throw an error and not proceed with an invalid configuration.
+      if (nativeBrokerInfo.broker.isBrokerAvailable === false) {
+        throw new Error(
+          [
+            "Broker for WAM was requested, and the plugin is configured, but the broker is not available.",
```

Comment:
I assume this code path will execute for macOS and Linux once those platforms are supported, so we should avoid using platform-specific broker names.

```suggestion
            "The broker was requested, and the plugin is configured, but the broker is unavailable.",
```